Sources/MountebankSwift/Models/Recreatable.swift

```diff
@@ -13,6 +13,8 @@ protocol Recreatable {
 private let delimiter = ",\n"
 private let whitespace = "    "
 private var indent = 0
+func increaseRecreatableIndent() { indent += 1 }
+func decreaseRecreatableIndent() { indent -= 1 }
```

Comment:
Needed to make sure indentation is right.

Yeah this is not ideal but it works. I'm fine with this "hack". 

The alternative would be to change the protocol from `var recreatable: String { get }` to `func recreatable(indent: Int)` which I had before at some point, but I didn't like that I needed to pass that property around everywhere.

Reply:
Using `func recreatable(indent: Int)` would be more tread safe.